lib/danger/danger_core/dangerfile_dsl.rb

```diff
@@ -13,6 +13,19 @@ module DSL
 
       attr_reader :env
 
+      # @!group Source Control Metadata
+      # Returns the name of the current SCM Provider being used.
+      # @return [Symbol] The name of the SCM Provider used for the active repository.
+      def scm_provider
+        if env.request_source && env.request_source.class == Danger::RequestSources::GitHub
+          :github
+        elsif env.request_source && env.request_source.class == Danger::RequestSources::GitLab
+          :gitlab
+        else
+          :unknown
+        end
+      end
```

Comment:
this this might be best on the `danger` plugin, it'll then get documented in the reference 👍